internal/router/router.go

```diff
@@ -104,5 +104,8 @@ func CreateAPIRouter(api api.IdlerAPI) *httprouter.Router {
 	router.GET("/api/idler/cluster", api.ClusterDNSView)
 	router.GET("/api/idler/cluster/", api.ClusterDNSView)
 
+	router.GET("/api/idler/reset/:namespace", api.Reset)
```

Comment:
why not POST ?

Reply:
Because I copied the about line and modified it. but you are right, POST more suitable